app/src/main/java/com/orange/ouds/app/ui/tokens/TokenPropertyHeader.kt

```diff
@@ -0,0 +1,239 @@
+ * or see the "LICENSE" file for more details.
+ *
+ * Software description: Android library of reusable graphical components
+ */
+
+package com.orange.ouds.app.ui.tokens
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.IntrinsicSize
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import com.orange.ouds.app.R
+import com.orange.ouds.core.theme.value
+import com.orange.ouds.core.utilities.OudsPreview
+import com.orange.ouds.foundation.utilities.BasicPreviewParameterProvider
+import com.orange.ouds.foundation.utilities.UiModePreviews
+import com.orange.ouds.theme.dashedBorder
+import com.orange.ouds.theme.tokens.OudsColorKeyToken
+import com.orange.ouds.theme.tokens.OudsSpaceKeyToken
+import com.orange.ouds.theme.tokens.OudsTypographyKeyToken
+
+@Composable
+fun GridHeader(modifier: Modifier = Modifier) {
+    Column(
+        modifier = modifier,
+        verticalArrangement = Arrangement.spacedBy(OudsSpaceKeyToken.Fixed.Medium.value)
+    ) {
+        val resourceIds = listOf(
+            R.drawable.il_tokens_grid_column_margin,
+            R.drawable.il_tokens_grid_min_width,
+            R.drawable.il_tokens_grid_max_width
+        )
+        resourceIds.forEach { resourceId ->
+            Image(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .background(OudsColorKeyToken.Background.Emphasized.value),
+                painter = painterResource(id = resourceId),
+                contentDescription = null
+            )
+        }
+    }
+}
+
+@Composable
+fun SpaceHeader(
+    spaceTokenProperty: TokenProperty<TokenCategory.Dimension.Space>,
+    modifier: Modifier = Modifier
+) {
+    Box(
+        modifier = modifier
+            .background(color = OudsColorKeyToken.Background.Emphasized.value)
+            .padding(all = OudsSpaceKeyToken.Fixed.Medium.value)
+    ) {
+        SpaceHeaderContent(spaceTokenProperty = spaceTokenProperty)
+    }
+}
+
+@Composable
+private fun SpaceHeaderContent(spaceTokenProperty: TokenProperty<TokenCategory.Dimension.Space>) {
+    val dashedBorderWidth = 2.dp
```

Comment:
Are you sure for the value? I cannot access the value on Figma but it seems better to me when set to `1.dp`.